### PR TITLE
Add legal mentions

### PR DESCRIPTION
**Pull request by ptbrowne**
Add a placeholder component that can be overrided to show a legal mention.

- Balance page
- Transactions Page
- Transfer page

I had a problem with the transactions header since adding the legal mention
would modify the header height. Since the header is fixed, the list beneath
it has a padding-top for its content to end up lower than the header. Thus,
the padding-top and the header height should be the same. It was previously
done manually in CSS. Unfortunately, I could not override the stylus file:
there were build errors due to MiniCSSExtractPlugin. An alternative solution
was to compute dynamically the height in JS and set the padding-top accordingly.

### CODE REVIEW

File: src/ducks/transactions/TransactionsPage.jsx [hunk 1]
```diff
@@ -94,6 +104,29 @@ class TransactionsPage extends Component {
...
+    window.removeEventListener('resize', this.handleResize)
+  }
+
+  handleResize() {
+    // eslint-disable-next-line
+    const headerNode = ReactDOM.findDOMNode(this.headerRef).firstChild
+    // eslint-disable-next-line
+    const listNode = ReactDOM.findDOMNode(this.listRef)
+
+    if (document.body.getBoundingClientRect().width < 768) {
```
Comment (Crash--) [inline src/ducks/transactions/TransactionsPage.jsx:122]: 768px is one of our breakpoint?

**Review by Crash-- — APPROVED** [review]

File: src/ducks/transactions/TransactionHeader.jsx [hunk 2]
```diff
@@ -84,6 +85,7 @@ const TransactionHeader = ({
           handleChangeMonth={handleChangeMonth}
           currentMonth={currentMonth}
         />
+        <LegalMention className="u-mt-1" />
```
[the remark this answers is not in the document]
Comment (ptbrowne) [reply src/ducks/transactions/TransactionHeader.jsx:88]: No, this is why there is the boolean `legaMention` that I'm going to rename as per your suggestion.

File: src/ducks/balance/BalanceHeader.jsx [hunk 3]
```diff
@@ -57,6 +58,11 @@ const BalanceHeader = ({
         </Delayed>
       )}
       <KonnectorUpdateInfo />
+      {LegalMention.active ? (
```
[the remark this answers is not in the document]
Comment (ptbrowne) [reply src/ducks/balance/BalanceHeader.jsx:61]: No, I added an explanation, it is a static property, it is useful when we need to add a wrapper around the LegalMention.

File: src/ducks/transactions/TransactionsPage.jsx (hunk 1 above)
Comment (ptbrowne) [reply src/ducks/transactions/TransactionsPage.jsx:122]: Yes 

> BP-small       =  768
> stylus/settings/breakpoints.styl

**Pull request merged**